cms/models.py

```diff
@@ -1694,6 +1695,16 @@ class CertificatePage(CourseProgramChildPage):
     CMS page representing a Certificate.
     """
 
+    class Placement(models.IntegerChoices):
```

Comment:
A more suitable name for this class would be `PartnerLogoPosition` or `PartnerLogoPlacement`.